src/parts/Media.tsx

```diff
@@ -174,6 +188,12 @@ export interface GenericMediaProps extends HTMLAttributes<HTMLDivElement> {
    * Received media (It can be an image or a video)
    */
   media?: MediaProps | VideoProps
+  image_srcset?: string
+  image_srcset_medium?: string
+  bgImageSrcSet?: string
```

Comment:
We have both `bgImageSrcSet` and `bg_image_srcset`. Is this intended?

Reply:
it was removed